kolibri/plugins/audio_mp3_render/assets/src/module.js

```diff
@@ -7,7 +7,7 @@ class AudioMP3Module extends ContentRendererModule {
     return AudioComponent;
   }
   get contentType() {
```

Comment:
Yes, it should be, an update in terminology to match the fact we can now render more than one content type with a single renderer!